### PR TITLE
Convert sample to use msbuild project

### PR DESCRIPTION
**Pull request by MichaelSimons**

### CODE REVIEW

File: dotnetapp/Dockerfile.build [hunk 1]
```diff
@@ -1,8 +1,8 @@
-FROM microsoft/dotnet:1.0-sdk-projectjson
+FROM microsoft/dotnet:1.0-sdk-msbuild
```
Comment (naamunds) [inline dotnetapp/Dockerfile.build:1]: Why don't we use `1.1-sdk-msbuild`?

File: dotnetapp/dotnetapp.csproj [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
```
Comment (naamunds) [inline dotnetapp/dotnetapp.csproj:1]: Can we assume that users will have MSBuild 15 on their machines?  My understanding is that it is associated with VS 2017, which is still just in RC. I wonder if using `ToolsVersion="14.0"` would work with both 14.0 and 15.0 and would thus be a safer choice.

**Review by naamunds — APPROVED** [review]

File: dotnetapp/dotnetapp.csproj (hunk 2 above)
Comment (MichaelSimons) [reply dotnetapp/dotnetapp.csproj:1]: Note this is for the dockerhub branch.  This is the sample we build an image for that is pushed to microsoft/dotnet.  The samples we expect users to widely view are in the master branch.  That being said, the necessary VS support only exists in VS 2017 AFAIK.

File: dotnetapp/Dockerfile.build (hunk 1 above)
Comment (MichaelSimons) [reply dotnetapp/Dockerfile.build:1]: This sample is using LTS.  I am not sure there is a reason to use current for this sample.

**Pull request merged**